base_ical/models/res_users.py

```diff
@@ -12,7 +12,7 @@ class ResUsers(models.Model):
 
     def _compute_ical_ids(self):
         domain = [("allowed_users_ids", "=", self.env.uid)]
-        self.write({"ical_ids": self.env["base.ical"].search(domain)})
```

Comment:
@fkantelberg please remind me why this is a write and not an assignment?

and why is this not

```python
for this in self:
    domain = [("allowed_users_ids", "=", this.id)]
    this.ical_ids = self.env["base.ical"].search(domain)
```

?

@jans23 please test if the above fixes your problem too, as this seems more correct to me

Reply:
Yes, it solves my issue.

BTW, it is version 15.0.